hudi-spark-datasource/hudi-spark/pom.xml

```diff
@@ -498,5 +498,11 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.mrpowers</groupId>
```

Comment:
Is this library widely used and stable enough for the testing?

Reply:
I think yes, it's enough for testing. It reduces execution time of each test-case for 3-4 seconds  (true case: from 37sec to 34sec; false case: from 23sec to 19sec).
But I'll remove it if you don't want to use this lib and reducing the number of inserts/updates is enough.

Reply:
We should remove it if it's not strictly required.

Reply:
removed